windows/hid.c

```diff
@@ -463,6 +467,12 @@ static int hid_internal_get_interface_number(const wchar_t* hardware_id)
 	return interface_number;
 }
 
+inline int hid_internal_match_device_id(unsigned short vendor_id, unsigned short product_id, unsigned short expected_vendor_id, unsigned short expected_product_id)
```

Comment:
interesting why MinGW doesn't inline this function (failed CI)
lets make it `static` - it wouldn't really affect performance, but at least one compiler will have less complains
```suggestion
static int hid_internal_match_device_id(unsigned short vendor_id, unsigned short product_id, unsigned short expected_vendor_id, unsigned short expected_product_id)
```